src/MongoDB/WriteConcern.c

```diff
@@ -91,7 +90,11 @@ PHP_METHOD(WriteConcern, __construct)
 	switch(ZEND_NUM_ARGS()) {
 		case 3:
 			if (Z_TYPE_P(journal) != IS_NULL) {
+#ifdef ZEND_ENGINE_3
+				mongoc_write_concern_set_journal(intern->write_concern, Z_TYPE_P(journal) == IS_TRUE);
```

Comment:
Addressed in https://github.com/mongodb/mongo-php-driver/pull/175/files#diff-2255238768a806cc38ff567bdb9bc2b5R94, where @derickr uses `zend_is_true()`.